docs/Streams/index.md

```diff
@@ -0,0 +1,73 @@
+# Streams
+
+A stream is a doubly linked list of bytes. It can be used for file input/output, DayZ serializers, hex conversion or string parsing. Once a stream is finished, it is recommended to close the stream either by letting the reference count equal zero or call the `Close` method.
+
+Streams can be manipulated with `CF_TextWriter`, and `CF_BinaryWriter`. Streams can be read with `CF_TextReader`, and`CF_BinaryReader`.
+
+## Files
+
+Use the `CF_FileStream` class to read and write files on the a file system. 
+
+```csharp
+string path = "$profile:test.txt";
+
+if (FileExist(path)) DeleteFile(path);
```

Comment:
The file mode should determine if the file is replaced or appended, so this line has no real value does it? I would remove it.